compiler/AST/ForallStmt.cpp

```diff
@@ -361,8 +373,18 @@ static VarSymbol* createAndAddIndexVar(AList& fIterVars, int idxPosition) {
   return result;
 }
 
+// Support different users of fsDestructureWhenSingleIdxVar().
+static inline VarSymbol* indexNodeToVarSymbol(BaseAST* index) {
```

Comment:
I generally avoid the term "AST Node" in favor of the AST type. In this case I'd call the routine `indexExprToVarSymbol`.